internal/k8s/controllers/reconciliation_test.go

```diff
@@ -465,21 +484,39 @@ var _ = Describe("FRRK8S Controller", func() {
 var _ = Describe("Layer2 Status Controller", func() {
 	Context("SetupWithManager", func() {
 		It("Should Reconcile correctly", func() {
-
-			statusObjName := fmt.Sprintf("%s-%s", testServiceName, layer2StatusReconciler.NodeName)
+			statusObjFetcherFunc := func() ([]v1beta1.ServiceL2Status, error) {
+				statusList := v1beta1.ServiceL2StatusList{}
+				err := k8sClient.List(context.TODO(), &statusList,
+					client.InNamespace(speakerNamespace),
```

Comment:
can you make this in all namespaces (just so we're sure we aren't creating unnecessary ones)